python/humbug/report.py

```diff
@@ -327,3 +327,21 @@ def _hook(exception_type, exception_instance, traceback):
             sys.excepthook = _hook
 
             self.is_excepthook_set = True
+
+    def setup_notebook_excepthook(self, tags: Optional[List[str]] = None):
+        """
+        Excepthook for ipython, works with jupiter notebook.
+        """
+        self.system_report(publish=True, tags=tags)
```

Comment:
@kompotkot : Let us remove the `system_report` publication.

Reply:
Done